### PR TITLE
Add max depth of company function to Company class

### PR DESCRIPTION
**Pull request by tedim52**
-Added max depth function w/tests to calculate the max depth of a company's hierarchy
-Also removed setup.sh, don't think it's necessary anymore

### CODE REVIEW

**Comment by tedim52** [comment]
Meant to do this in seperate PR's my bad.

-Added __create_hierarchy function to Company class w/test case
 The function recursively creates a dict representation of the company hierarchy and allows the company hierarchy to be outputted as a json file as shown, this format follows the D3 collapsible tree documentation of the required input data to create a collapsible tree using the library. Hopefully, this makes putting it on the frontend a lot easier!
https://github.com/d3/d3-hierarchy/blob/master/README.md#hierarchy

Screen Shot 2020-07-22 at 4 49 31 PM

**Comment by tedim52** [comment]
-Added simple notebook file to visualize distance vs. usage similarity regression when data is ready
Screen Shot 2020-07-22 at 5 19 10 PM

File: core/company.py [hunk 1]
```diff
@@ -18,16 +21,89 @@ class Company:
     def __init__(self, employees):
         """ Creates Google company.
 
-        Calls __create_company method that implements manager hierarchy tree based on the list of
+        Calls __create_directory method that implements manager hierarchy tree based on the list of
```
Comment (DeanAlvarez) [inline core/company.py:24]: NIT: I suspect this is over 80 characters

File: core/company_test.py [hunk 2]
```diff
@@ -5,46 +5,93 @@
...
+        #             0
+        #     1       2       8
+        #                   7
 
-    def test_finds_employee(self):
-        #Test that the correct employee is found
-        self.assertEqual(self.actual_employee, self.company.search(14))
+        self.actual_hierarchy = { "Employee": -100, "Reports": [ {"Employee": 0, "Reports":[ {"Employee": 1},
+                                                                                {"Employee": 2 },
+                                                                                {"Employee": 8, "Reports":[ {"Employee": 7 }]}
```
Comment (DeanAlvarez) [inline core/company_test.py:31]: This also looks over 80 characters

**Review by DeanAlvarez — APPROVED** [review]
LGTM. I'd check to make sure everything is under 80 characters but besides that seems good

**Pull request merged**